IRremoteESP8266.cpp

```diff
@@ -482,7 +482,7 @@ void ICACHE_FLASH_ATTR IRsend::sendGree(unsigned char data[]) {
 
   // Footer #2
   mark(GREE_BIT_MARK);
-  space(GREE_MSG_SPACE);
```

Comment:
I deliberately changed your existing `space(0)` to `space(GREE_MSG_SPACE)` as it is good practice for an appropriate gap to exist after a command, so that a subsequent message can't be sent and confuse the receiver.

However, if this is breaking something, let me know. It really shouldn't.